Sources/SwiftProtobuf/ProtobufBinaryTypes.swift

```diff
@@ -679,6 +750,7 @@ public protocol ProtobufBinaryMessageBase: ProtobufMessageBase {
     // Serialize to protobuf
     func serializeProtobuf() throws -> Data
     func serializeProtobufBytes() throws -> [UInt8]
```

Comment:
Related:  I've been planning to drop whichever of `Data` or `[UInt8]` is significantly slower.  If you're confident that the `Data` approach is really going to be the winner, we could start stripping out the `[UInt8]` versions.

(If we found that `Data` was slower, we would still need to support `Data` at the API boundary for interoperability reasons, but could stick with `[UInt8]` internally.)